src/location-registry/models/location.py

```diff
@@ -8,12 +8,13 @@
 if os.getenv('FLASK_ENV') == 'production':
     MONGO_URI = os.getenv('PROD_MONGO_URI')
```

Comment:
> Good stuff...
> Just to avoid code duplication, is there a way we can have these configuration statements in one place (App's start script perhaps) and reuse them?

This [PR](https://github.com/airqo-platform/AirQo-api/pull/194) attempts to take care of configurations